pset-functions.js

```diff
@@ -175,8 +235,21 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
-// Add your code above this line
+function palindrome(str) {
 
+    var len = str.length;
+    var mid = Math.floor(len/2);
```

Comment:
@deedeemedina interesting algorithm, great job getting it to work. [Check out the solution](https://github.com/jsd20191008/hw-01-functions-solution/blob/master/pset-functions.js#L246) for an alternative approach to this problem